packages/opencensus-exporter-stackdriver/test/test-stackdriver-monitoring.ts

```diff
@@ -81,35 +69,23 @@ function assertMetricDescriptor(
   assert.strictEqual(metricDescriptor.unit, view.measure.unit);
 }
 
+
 /**
  * Asserts TimeSeries' values given its originating view.
- * @param TimeSeries The TimeSeries to be asserted.
- * @param view The originating view.
- * @param measurement The originating measurement.
- * @param projectId The project ID from the Stackdriver Monitoring.
+ * @param {TimeSeries} TimeSeries The TimeSeries to be asserted.
+ * @param {View} view The originating view.
+ * @param {Measurement} measurement The originating measurement.
+ * @param {string} projectId The project ID from the Stackdriver Monitoring.
+ * @param {MetricKind} metricKind the metricKind of metric.
+ * @param {ValueType} valueType the valueType of metric.
+ * @param {value} value the value of point.
  */
 function assertTimeSeries(
     timeSeries: TimeSeries, view: View, measurement: Measurement,
-    projectId: string,
+    projectId: string, metricKind: MetricKind, valueType: ValueType, value: {},
```

Comment:
Please define [one statement per line](https://google.github.io/styleguide/jsguide.html#formatting-one-statement-perline)

Reply:
This is by ```npm run fix```.